src/plugins/muc-views/templates/occupant.js

```diff
@@ -13,6 +13,72 @@ const i18n_occupant_hint = /** @param {MUCOccupant} o */(o) => {
     return __('Click to mention %1$s in your message.', o.get('nick'));
 }
 
+let badges_definitions; // will be initialized at first call (to be sure that the __ function is correctly loaded).
+
+/**
+ * Inits badges definitions.
+ * For short labels, it will use the label first letter. If there is ambigous short labels, it will try to add up to 4 letters.
+ * Letters will be uppercase.
+ */
+function init_badges_definitions () {
+    badges_definitions = {}
+    badges_definitions['owner'] = {
+        label: __('Owner'),
+        classname: 'badge-groupchat'
+    };
+    badges_definitions['admin'] = {
+        label: __('Admin'),
+        classname: 'badge-info'
+    };
+    badges_definitions['member'] = {
+        label: __('Member'),
+        classname: 'badge-info'
+    };
+    badges_definitions['moderator'] = {
+        label: __('Moderator'),
+        classname: 'badge-info'
+    };
+    badges_definitions['visitor'] = {
+        label: __('Visitor'),
+        classname: 'badge-secondary'
+    };
+
+    // And now we must compute unique short labels.
+    let seen;
+    for (
+        let currentLength = 1;
```

Comment:
I know this is different than how 99% of JavaScript projects write variables, and it's a habit I brought over from the Python world, but in Converse we write variables with underscores.

So this should be `current_length`.

I've thought about changing this rule, but it would be a lot of work to rename all variables!
At the same time, having a mix of different styles for writing variables is also not great.

Reply:
Oops. I did it for other variables and functions, i just missed this one. I just forced push the fix (and rebased on master to fix the CI).